extensions/wikia/EmbeddableDiscussions/EmbeddableDiscussionsController.class.php

```diff
@@ -31,6 +31,7 @@ public static function render( $input, array $args, Parser $parser, PPFrame $fra
 		$showLatest = filter_var( $args['latest'], FILTER_VALIDATE_BOOLEAN );
 		$itemCount = empty( $args['size'] ) ? self::ITEMS_DEFAULT : intval( $args['size'] );
 		$columns = empty( $args['columns'] ) ? self::COLUMNS_DEFAULT : intval( $args['columns'] );
+		$category = empty( $args['category'] ) ? $wgCityId : ( new DiscussionsCategoryModel( $wgCityId ) )->getCategoryId( $args['category'] );
```

Comment:
If you leave off the `category`/`forumId` in the API call, e.g.:
```
https://services.wikia.com/discussion/{cityId}/threads
```
You'll get threads across all categories.  If you specify the `General` category (i.e. forumId={cityId}), you'll only get posts from there.  Depending on your product requirements, maybe you want this, but it seems like you might be better served if this controller could get you posts from a specific category, or all posts if a category isn't specified.